spec/NestHydrationJS/nest.spec.js

```diff
@@ -1072,4 +1072,46 @@ describe('NestHydrationJS', function () {
 			});
 		});
 	});
+
+	it('Should not break on null data with complex nesting', function () {
+		const rows = [
```

Comment:
Thanks. 

I fixed that in my branch, but the PR isn't updated automatically. I'll do that manually.